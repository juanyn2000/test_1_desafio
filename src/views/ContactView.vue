<template>
    <h1>Contact</h1>
</template>

<script>
export default {
    name: 'ContactView',
    data() {
        return {};
    }
};
</script>

<style scoped>
</style>