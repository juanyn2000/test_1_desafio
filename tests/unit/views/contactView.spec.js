import { shallowMount } from "@vue/test-utils";
import ContactView from "@/views/ContactView.vue";

describe("Componente Contador.vue", () => {
  test("Validación de match con el snapshot", () => {
    //Selección el Sujeto de pruebas
    const wrapper = shallowMount(ContactView);
    //Aserción
    expect(wrapper.html()).toMatchSnapshot();
  });
});

