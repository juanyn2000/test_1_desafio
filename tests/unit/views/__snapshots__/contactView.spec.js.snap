// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente Contador.vue Validaci√≥n de match con el snapshot 1`] = `<h1>Contact</h1>`;
